src/user-guide/installing-workflows.rst

```diff
@@ -238,10 +291,11 @@ Looking at the directory structure that has been created
 2. Symlinking of Directories
```

Comment:
I'm not totally sure that this and number 1 should be numbered, at least not explicitly.